include/aspect/postprocess/visualization/viscosity_tensor.h

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Is this file a part of the coookbook, or do you want it to be usable in a broader context? I believe it's the former, and in that case you should probably move this .h and the corresponding .cc file into the cookbook folder.